csp_nonce/middleware.py

```diff
@@ -36,7 +36,7 @@ def process_response(self, request, response):
                 return response
 
             has_nonce = nonce_exists(response)
-            if has_nonce:
+            if bool(has_nonce):
```

Comment:
`has_nonce` sounds like it should be a bool as it's returned from `nonce_exist`, casting to bool doesn't look right in here.